app/src/main/java/com/brandon/forzenbook/view/MainActivity.kt

```diff
@@ -44,8 +43,8 @@ class MainActivity : ComponentActivity() {
                                     onDismiss = {
                                         topLevelViewModel.dismissNotification()
                                     },
-                                    onSubmit = { username, password ->
-                                        topLevelViewModel.login(username, password)
+                                    onSubmit = { username, code ->
+                                        topLevelViewModel.login(username, code)
```

Comment:
Remember that the VM receives events from the view like "loginClicked", UI related event names

Reply:
Changed to Login Clicked, will do a general pass on these in the view level rework to handle the code.